src/plugin/file/kml/kmlnodelayerui.js

```diff
@@ -1092,6 +1110,27 @@ plugin.file.kml.KMLNodeLayerUICtrl.prototype.isFeatureDynamic = function() {
 };
 
 
+/**
+ * If the feature is fillable, which means it should show fill controls
+ *
+ * @return {boolean}
+ * @export
+ */
+plugin.file.kml.KMLNodeLayerUICtrl.prototype.isFeatureFillable = function() {
+  var features = this.getFeatures();
+  var feature = features.length > 0 ? features[0] : null;
+  if (feature) {
+    var geometry = feature.getGeometry();
+
+    if (geometry instanceof ol.geom.Polygon || geometry instanceof ol.geom.MultiPolygon) {
```

Comment:
This can be replaced with `os.geo.isGeometryPolygonal`. We also prefer checking the geometry's `getType` value over using `instanceof`.

Reply:
This has been updated.